code/modules/food_and_drinks/food/snacks_pastry.dm

```diff
@@ -189,6 +189,28 @@
 	list_reagents = list("nutriment" = 4, "omnizine" = 3)
 	tastes = list("meat" = 2, "dough" = 2, "laziness" = 1)
 	foodtype = GRAIN
+	var/cooked_time = 0
+
+/obj/item/reagent_containers/food/snacks/donkpocket/warm/Initialize()
+	. = ..()
+	START_PROCESSING(SSprocessing, src)
```

Comment:
Make this a timer or something

Reply:
alternatively, make the bagged ones not process() but make the heated ones get cold sooner

Reply:
this has a time that is different based on loc, not just 1 static time. also, probs